broker-core/src/test/java/io/zeebe/broker/workflow/WorkflowInstanceTokenTest.java

```diff
@@ -0,0 +1,411 @@
+/*
```

Comment:
The concept of the defer and spawn token is more internal or? In the tests we are not really seeing if really a token was spawned or deferred. Maybe it makes sense to rename the tests? Do we need to test the different Intermediate events like timer and messages? In the end they use the same logic or? I mean if we add more intermediate events do we need to test them here as well?